test/unit/operations/abstract_command.test.ts

```diff
@@ -49,12 +49,12 @@ describe('class CommandOperation', () => {
   });
 
   context('when a server is created', () => {
-    it('calls server.commandAsync when executeCommand is invoked', async () => {
+    it('calls executeCommand, which calls server.commandAsync, when executeCommandCallback is invoked', async () => {
```

Comment:
commandAsync would be a separate test, we still want coverage for executeCommand being invoked